### PR TITLE
[1.19][go1.15] Update to go1.15.12

### PR DESCRIPTION
**Pull request by cpanato**
**What type of PR is this?**

/kind feature
/area dependency

**What this PR does / why we need it**:

- Update to go1.15.12

Tracking issue: https://github.com/kubernetes/release/issues/2060
Slack thread: https://kubernetes.slack.com/archives/C2C40FMNF/p1620388464296500

/assign @justaugustus @mkumatag @liggitt @dims 
cc: @kubernetes/release-engineering 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- Kubernetes is now built using go1.15.12
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/images/sample-apiserver/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 The Kubernetes Authors.
```
Comment (liggitt) [inline test/images/sample-apiserver/Makefile:1]: is this new file expected / needed?

Comment (cpanato) [reply test/images/sample-apiserver/Makefile:1]: no, removing

Comment (cpanato) [reply test/images/sample-apiserver/Makefile]: done, thanks for the review!

**Pull request merged**